01_Data/src/layers/sequelize/model/ChargingStationSecurityInfo.ts

```diff
@@ -0,0 +1,23 @@
+import { Column, Model, Table, DataType } from 'sequelize-typescript';
+import { Namespace } from '@citrineos/base';
+
+/**
+ * Represents the security information found on a particular charging station.
+ */
+@Table
+export class ChargingStationSecurityInfo extends Model {
```

Comment:
Why exactly is it required that we store this information?

Reply:
this is stored because the publicKey isn't always sent in the signed meter value -- if it's missing, we'll need some place to refer to a previously sent public key for the charging station

Reply:
Is it really necessary to store the key in the first place?

When charger sends key in the meter values, don't we reject the request if it doesn't match the key CSMS has configured?

By this logic, then the key that we store in the DB will always be what is configured in the CSMS. In which case, the entire DB will consist of redundant data.

Reply:
yup indeed currently, the key should always match what's in the CSMS, rather than needing to refer to this object. however, down the line, chargers may be able to have different configured keys, so this is really a setup for the future 🤞